models/Movie.model.js

```diff
@@ -0,0 +1,13 @@
+const { mongoose, Schema, model } = require('mongoose');
+
+const movieSchema = new Schema({
+	title: String,
+	genre: String,
+	plot: String,
+	cast: {
+		type: mongoose.Schema.Types.ObjectId,
```

Comment:
I would go with just that one, as you already using it above so you would't need to import mongoose too

```suggestion
		type: Schema.Types.ObjectId,
```